kasa/httpclient.py

```diff
@@ -117,6 +117,21 @@ async def post(
                     response_data = await resp.read()
                     if return_json:
                         response_data = json_loads(response_data.decode())
+                else:
+                    response_data = await resp.read()
+                    _LOGGER.debug(
+                        "Device %s received status code %s with response %s",
+                        self._config.host,
+                        resp.status,
+                        str(response_data),
+                    )
```

Comment:
This could be moved before the status code code to make it cover both paths.

Reply:
That's done. Also thought to de-indent after the read